src/prebid.js

```diff
@@ -514,7 +504,9 @@ $$PREBID_GLOBAL$$.requestBids = function ({ bidsBackHandler, timeout, adUnits, a
   }
 
   //set timeout for all bids
-  setTimeout(bidmanager.executeCallback, cbTimeout);
+  const timedOut = true;
+  const timeoutCallback = bidmanager.executeCallback.bind(bidmanager, timedOut);
+  setTimeout(timeoutCallback, cbTimeout);
```

Comment:
`setTimeout` must be kept to ensure that the timeout is recorded even if there is no response after the limit - see f4ed5c2cbefaec6ea5b0cdda1217ba55d1288d82 commit message
I will add back the previous method however, so that whichever is encountered first will be used.